frontend/src/__tests__/cypress/cypress/pages/pipelines/pipelinesGlobal.ts

```diff
@@ -132,13 +132,84 @@ class ConfigurePipelineServerModal extends Modal {
   findDatabaseInput() {
     return this.find().findByTestId('field Database');
   }
+
+  configurePipelineServer(projectName: string) {
+    pipelinesGlobal.findConfigurePipelineServerButton().should('be.enabled');
+    pipelinesGlobal.findConfigurePipelineServerButton().click();
+
+    this.findAwsKeyInput().type('test-aws-key');
+    this.findAwsSecretKeyInput().type('test-secret-key');
+    this.findEndpointInput().type('https://s3.amazonaws.com/');
+    this.findRegionInput().should('have.value', 'us-east-1');
+    this.findBucketInput().type('test-bucket');
+    this.findSubmitButton().should('be.enabled');
+    this.findSubmitButton().click();
+
+    cy.wait('@createSecret').then((interception) => {
```

Comment:
This function has a soft dependency on named interceptors being present.
As a page object, it should be reusable for multiple scenarios and should not make any assumptions about setup.

I think you're better off sharing these kinds of assertions in a shared test utility.